src/gui/tray/activitylistmodel.h

```diff
@@ -48,6 +48,8 @@ class ActivityListModel : public QAbstractListModel
         ActionIconRole = Qt::UserRole + 1,
         AccountRole,
         ObjectTypeRole,
+        ObjectIdRole,
+        ObjectNameRole,
```

Comment:
looks like you are replacing `AbsolutePathRole` by `ObjectIdRole` can you remove it ?

Reply:
AbsolutePathRole is still in the enum

Reply:
Sorry I misunderstood you, I thought you were asking if it had been removed. I will remove it

Reply:
Okay, removed